Jenkinsfile

```diff
@@ -58,26 +58,28 @@ node('linux') {
 
 		// HACK: Recommended approach for getting command output is writing to and then reading a file.
 		sh 'mkdir -p tmp'
-		sh 'go list . > tmp/GO_LIST'
-		// HACK: Jump hurdle to get at actual PR commit rather than merge commit
-		sh 'git rev-parse HEAD | git log --pretty=%P -n 1 | cut -c1-40 > tmp/ACTUAL_COMMIT'
 		sh 'git describe --all > tmp/GIT_BRANCH'
-		go_list = readFile('tmp/GO_LIST').trim()
-		actual_commit = readFile('tmp/ACTUAL_COMMIT').trim()
+		sh 'git rev-parse HEAD > tmp/GIT_COMMIT'
+		sh 'go list . > tmp/GO_LIST'
 		git_branch = readFile('tmp/GIT_BRANCH').trim()
-		// convert 'github.com/deis/controller-sdk-go' to 'github.com/${env.CHANGE_AUTHOR}/controller-sdk-go'
-		go_repo = go_list.replace('deis', env.CHANGE_AUTHOR)
+		git_commit = readFile('tmp/GIT_COMMIT').trim()
+		go_repo = readFile('tmp/GO_LIST').trim()
+
+		if (git_branch != "remotes/origin/master") {
+			// HACK: Jump hurdle to get at actual PR commit rather than merge commit
```

Comment:
@arschles created in https://github.com/deis/controller-sdk-go/issues/45, will update the comment here as well.